### PR TITLE
[DOCS-6865] Alfresco Mobile Workspace 1.4

### PR DESCRIPTION
**Pull request by anxumalo**
Create PR since the feature updates in PR #784 were merged before the UA review was completed.

### CODE REVIEW

File: mobile-workspace/latest/using/index.md [hunk 1]
```diff
@@ -133,3 +133,50 @@ You can move files within the Mobile Workspace.
...
+
+## Tasks
+
+You can view, filter, complete, and add comments to tasks while logged into the Mobile Workspace.
+
+### Tasks Overview
+
+The **Tasks** button is located at the bottom between **Favorites** and **Offline**. The Tasks list loads active tasks assigned to you or created by you. Task priority colors are set as the following: Low (green), Medium (orange), and High (red). Tasks will refresh when the user scrolls down to the end of the task list or when the refresh button is pressed. The refresh button will also reset filters.
+
+* A Dark mode for Tasks is set within user Settings under Appearance.
```
Comment (oliveralfresco2018) [inline mobile-workspace/latest/using/index.md:145]:
```suggestion
* A Dark mode for Tasks can be set within user Settings under Appearance.
```

File: mobile-workspace/latest/using/index.md [hunk 2]
```diff
@@ -133,3 +133,50 @@ You can move files within the Mobile Workspace.
...
+
+1. Select a filter chip.
+
+2. Follow the directions for each filter as follows:
+
+    * A Status filter can be set by selecting “Active” or “Completed” and press **Apply**. The Status filter chip label will show the active filter, so “Completed” instead of “Status”.
+    * Press **Due date** to set a custom, date-ranged filter. “From” and “To” start and end fields set using a calendar date picker. Only one date needs to be set.
+    * Press **Task Name** to filter tasks by name. Type your search in a text field. This filter does not apply to priority and is not case sensitive.
+
+3. Press **Apply** to set the filter and **Reset** to reset the filter to default. The **X** in top right of the filter UI will close the window and return to the Task list.
```
Comment (oliveralfresco2018) [inline mobile-workspace/latest/using/index.md:161]:
```suggestion
3. Press **Apply** to set the filter and **Reset** to reset the filter to default. The **X** in the top right of the filter UI will close the window and return to the Task list.
```

File: mobile-workspace/latest/using/index.md [hunk 3]
```diff
@@ -133,3 +133,50 @@ You can move files within the Mobile Workspace.
...
+Attached files is a section below Comments that shows the number of attached files to the task. The link **View all** displays an Attached files page when there are more than four attachments. Clicking an attachment will download the file to preview with a popup and **Cancel** option.
+
+### Complete Tasks
+
+You will be able to complete tasks if the task is assigned to you.
+
+1. Open a task from the Task list. At the bottom of the task is a **Complete** Floating Action Button.
+
+2. Press the **Complete** button. A popup is displayed:
+    “Are you sure you want to complete the task? You will no longer be able to make any changes.” A completed task can be viewed, but enters a read-only mode. No comments or attachments can be added to a completed task.
```
Comment (oliveralfresco2018) [inline mobile-workspace/latest/using/index.md:180]:
```suggestion
    “Are you sure you want to complete the task? You will no longer be able to make any changes.” A completed task can be viewed, but in read-only mode. No comments or attachments can be added to a completed task.
```

**Review by oliveralfresco2018 — CHANGES_REQUESTED** [review]
Minor suggestions, cheers

File: mobile-workspace/latest/using/index.md (hunk 3 above)
Comment (anxumalo) [reply mobile-workspace/latest/using/index.md]: I left this as-is

**Review by oliveralfresco2018 — APPROVED** [review]
LGTM

**Pull request merged**